app/api/srch/search.rb

```diff
@@ -103,7 +103,10 @@ class Search < Grape::API
               DocResult.new(
                 doc_type: 'USERS',
```

Comment:
```suggestion
                  doc_type: 'USERS',
```

Reply:
I think codeclimate error will get resolved if you'll add two more spaces from 104-109. Would you like to try it? Although, the work here is done and we can merge it with the codeclimate error too. Thanks!

Reply:
Oh.  I hadn't noticed that.  I'll make the change.